source/overloading.tex

```diff
@@ -2680,16 +2699,9 @@
 or, if not that,
 
 \item
-\tcode{S1}
-and
-\tcode{S2}
-differ only in their qualification conversion and yield similar types
-\tcode{T1}
-and
-\tcode{T2}\iref{conv.qual}, respectively, and the cv-qualification signature of type
-\tcode{T1}
-is a proper subset of the cv-qualification signature of type
-\tcode{T2}
+\tcode{S1} and \tcode{S2} differ only in their qualification conversion and
+yield similar types \tcode{T1} and \tcode{T2}\iref{conv.qual}, respectively,
+where \tcode{T1} can be converted to \tcode{T2} by a qualification conversion
```

Comment:
Paper wants a reference to [conv.qual] here.  Can it be added to the 1st "qualification conversion" in the sentence instead please?  This is the first use in this section (the previous reference for "qualification conversion" is in [over.match.conv]).

Reply:
cross-reference moved to after "qualification conversion"